kairon/async_callback/main.py

```
@@ -1,42 +1,62 @@
-from fastapi.middleware.cors import CORSMiddleware
-from fastapi.middleware.gzip import GZipMiddleware
-from fastapi.responses import JSONResponse
+from kairon.async_callback.router import pyscript_callback
+
+from blacksheep import Application, Request, Response
+from blacksheep.server.responses import json as JSONResponse
 from loguru import logger
 from secure import StrictTransportSecurity, ReferrerPolicy, ContentSecurityPolicy, XContentTypeOptions, Server, \
     CacheControl, Secure, PermissionsPolicy
-
 from kairon.api.models import Response
```

Ruff (0.8.2)

Redefinition of unused `Response` from line 11

(F811)